src/importlinter/domain/imports.py

```diff
@@ -27,6 +27,9 @@ def __init__(self, name: str) -> None:
         """
         self.name = name
 
+    def has_wildcard_expression(self) -> bool:
```

Comment:
I don't think this commit belongs in this PR, correct?

It would be interesting to support module expressions in standalone contracts, but perhaps that's left for another day!